presto-main/src/main/java/com/facebook/presto/sql/planner/optimizations/PreferredProperties.java

```diff
@@ -299,6 +283,28 @@ public String toString()
                     .add("partitioningProperties", partitioningProperties)
                     .toString();
         }
+
+        public Global translate(Function<VariableReferenceExpression, Optional<VariableReferenceExpression>> translator)
```

Comment:
Why to move these methods?

Reply:
I made them private because they were public, but only called internally. I moved them to the bottom because on projects I've been on in the past the private methods that aren't part of the public interface go last.

Before merging I wouldn't squash the commit for this change in with the rest.